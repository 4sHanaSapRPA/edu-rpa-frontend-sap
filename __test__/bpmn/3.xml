<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0jl7jys" targetNamespace="http://bpmn.io/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="14.0.0">
  <bpmn:process id="Process_04og7m0" isExecutable="false">
    <bpmn:startEvent id="StartEvent_1mxxylx">
      <bpmn:outgoing>Flow_12ctyks</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:task id="Activity_0pkjxtm" name="Open Browser">
      <bpmn:incoming>Flow_12ctyks</bpmn:incoming>
      <bpmn:outgoing>Flow_1ufgvsl</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_12ctyks" sourceRef="StartEvent_1mxxylx" targetRef="Activity_0pkjxtm" />
    <bpmn:task id="Activity_1536xo5" name="Get Title Assign to ${title}">
      <bpmn:incoming>Flow_1ufgvsl</bpmn:incoming>
      <bpmn:outgoing>Flow_0c0pvmq</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_1ufgvsl" sourceRef="Activity_0pkjxtm" targetRef="Activity_1536xo5" />
    <bpmn:exclusiveGateway id="Gateway_0sc92or">
      <bpmn:incoming>Flow_0c0pvmq</bpmn:incoming>
      <bpmn:outgoing>Flow_1u5v65y</bpmn:outgoing>
      <bpmn:outgoing>Flow_0q637w1</bpmn:outgoing>
    </bpmn:exclusiveGateway>
    <bpmn:sequenceFlow id="Flow_0c0pvmq" sourceRef="Activity_1536xo5" targetRef="Gateway_0sc92or" />
    <bpmn:task id="Activity_1gusysf" name="Log Screen">
      <bpmn:incoming>Flow_1u5v65y</bpmn:incoming>
      <bpmn:outgoing>Flow_0acg68n</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_1u5v65y" name="&#39;Element&#39; in ${title}" sourceRef="Gateway_0sc92or" targetRef="Activity_1gusysf" />
    <bpmn:task id="Activity_0m9hqf4" name="Log Screen">
      <bpmn:incoming>Flow_0q637w1</bpmn:incoming>
      <bpmn:outgoing>Flow_0xd1c08</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_0q637w1" name="else" sourceRef="Gateway_0sc92or" targetRef="Activity_0m9hqf4" />
    <bpmn:exclusiveGateway id="Gateway_1w9pebz">
      <bpmn:incoming>Flow_1r4gg47</bpmn:incoming>
      <bpmn:incoming>Flow_03hvnx5</bpmn:incoming>
      <bpmn:outgoing>Flow_0acsgi9</bpmn:outgoing>
    </bpmn:exclusiveGateway>
    <bpmn:sequenceFlow id="Flow_0acg68n" sourceRef="Activity_1gusysf" targetRef="Activity_173a9c7" />
    <bpmn:task id="Activity_012ckfs" name="Close Browser">
      <bpmn:incoming>Flow_0acsgi9</bpmn:incoming>
      <bpmn:outgoing>Flow_0piu48d</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_0acsgi9" sourceRef="Gateway_1w9pebz" targetRef="Activity_012ckfs" />
    <bpmn:endEvent id="Event_10jy91v">
      <bpmn:incoming>Flow_0piu48d</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:sequenceFlow id="Flow_0piu48d" sourceRef="Activity_012ckfs" targetRef="Event_10jy91v" />
    <bpmn:task id="Activity_173a9c7" name="Click &#34;Dịch vụ Internet&#34;">
      <bpmn:incoming>Flow_0acg68n</bpmn:incoming>
      <bpmn:outgoing>Flow_1r4gg47</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_1r4gg47" sourceRef="Activity_173a9c7" targetRef="Gateway_1w9pebz" />
    <bpmn:task id="Activity_0p2itoz" name="Click &#34;Thảo luận&#34;">
      <bpmn:incoming>Flow_1m11vkb</bpmn:incoming>
      <bpmn:outgoing>Flow_08g2s7f</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_0xd1c08" sourceRef="Activity_0m9hqf4" targetRef="Gateway_12fobzq" />
    <bpmn:exclusiveGateway id="Gateway_12fobzq">
      <bpmn:incoming>Flow_0xd1c08</bpmn:incoming>
      <bpmn:outgoing>Flow_1m11vkb</bpmn:outgoing>
      <bpmn:outgoing>Flow_1eahtb9</bpmn:outgoing>
    </bpmn:exclusiveGateway>
    <bpmn:sequenceFlow id="Flow_1m11vkb" sourceRef="Gateway_12fobzq" targetRef="Activity_0p2itoz" />
    <bpmn:task id="Activity_072vuzz" name="Break">
      <bpmn:incoming>Flow_1eahtb9</bpmn:incoming>
      <bpmn:outgoing>Flow_16hyvza</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_1eahtb9" sourceRef="Gateway_12fobzq" targetRef="Activity_072vuzz" />
    <bpmn:task id="Activity_1juwgjw" name="Break">
      <bpmn:incoming>Flow_19945vu</bpmn:incoming>
      <bpmn:outgoing>Flow_03hvnx5</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_03hvnx5" sourceRef="Activity_1juwgjw" targetRef="Gateway_1w9pebz" />
    <bpmn:sequenceFlow id="Flow_16hyvza" sourceRef="Activity_072vuzz" targetRef="Activity_1az9oha" />
    <bpmn:task id="Activity_1az9oha" name="Click &#34;Thảo luận&#34;">
      <bpmn:incoming>Flow_08g2s7f</bpmn:incoming>
      <bpmn:incoming>Flow_16hyvza</bpmn:incoming>
      <bpmn:outgoing>Flow_19945vu</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_08g2s7f" sourceRef="Activity_0p2itoz" targetRef="Activity_1az9oha" />
    <bpmn:sequenceFlow id="Flow_19945vu" sourceRef="Activity_1az9oha" targetRef="Activity_1juwgjw" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_04og7m0">
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1mxxylx">
        <dc:Bounds x="152" y="162" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_0pkjxtm_di" bpmnElement="Activity_0pkjxtm">
        <dc:Bounds x="280" y="140" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_1536xo5_di" bpmnElement="Activity_1536xo5">
        <dc:Bounds x="470" y="140" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Gateway_0sc92or_di" bpmnElement="Gateway_0sc92or" isMarkerVisible="true">
        <dc:Bounds x="615" y="155" width="50" height="50" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_0mbxrw9" bpmnElement="Activity_1gusysf">
        <dc:Bounds x="750" y="80" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_1exvj2h" bpmnElement="Activity_0m9hqf4">
        <dc:Bounds x="710" y="210" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_0eltz4d" bpmnElement="Activity_012ckfs">
        <dc:Bounds x="1210" y="140" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_10jy91v_di" bpmnElement="Event_10jy91v">
        <dc:Bounds x="1402" y="162" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_1d9ztt6" bpmnElement="Activity_173a9c7">
        <dc:Bounds x="920" y="80" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_072vuzz_di" bpmnElement="Activity_072vuzz">
        <dc:Bounds x="710" y="470" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_01ka87d" bpmnElement="Activity_1az9oha">
        <dc:Bounds x="990" y="340" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_06rx3y5" bpmnElement="Activity_0p2itoz">
        <dc:Bounds x="830" y="340" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Gateway_12fobzq_di" bpmnElement="Gateway_12fobzq" isMarkerVisible="true">
        <dc:Bounds x="735" y="355" width="50" height="50" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_0bnioio" bpmnElement="Gateway_1w9pebz" isMarkerVisible="true">
        <dc:Bounds x="1095" y="135" width="50" height="50" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="BPMNShape_051pi78" bpmnElement="Activity_1juwgjw">
        <dc:Bounds x="990" y="220" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="Flow_12ctyks_di" bpmnElement="Flow_12ctyks">
        <di:waypoint x="188" y="180" />
        <di:waypoint x="280" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1ufgvsl_di" bpmnElement="Flow_1ufgvsl">
        <di:waypoint x="380" y="180" />
        <di:waypoint x="470" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0c0pvmq_di" bpmnElement="Flow_0c0pvmq">
        <di:waypoint x="570" y="180" />
        <di:waypoint x="615" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1u5v65y_di" bpmnElement="Flow_1u5v65y">
        <di:waypoint x="640" y="155" />
        <di:waypoint x="640" y="120" />
        <di:waypoint x="750" y="120" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="640" y="86" width="57" height="27" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0q637w1_di" bpmnElement="Flow_0q637w1">
        <di:waypoint x="640" y="205" />
        <di:waypoint x="640" y="250" />
        <di:waypoint x="710" y="250" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="645" y="225" width="21" height="14" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0acg68n_di" bpmnElement="Flow_0acg68n">
        <di:waypoint x="850" y="120" />
        <di:waypoint x="920" y="120" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0acsgi9_di" bpmnElement="Flow_0acsgi9">
        <di:waypoint x="1145" y="160" />
        <di:waypoint x="1178" y="160" />
        <di:waypoint x="1178" y="180" />
        <di:waypoint x="1210" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0piu48d_di" bpmnElement="Flow_0piu48d">
        <di:waypoint x="1310" y="180" />
        <di:waypoint x="1402" y="180" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1r4gg47_di" bpmnElement="Flow_1r4gg47">
        <di:waypoint x="1020" y="120" />
        <di:waypoint x="1058" y="120" />
        <di:waypoint x="1058" y="160" />
        <di:waypoint x="1095" y="160" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0xd1c08_di" bpmnElement="Flow_0xd1c08">
        <di:waypoint x="760" y="290" />
        <di:waypoint x="760" y="355" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1m11vkb_di" bpmnElement="Flow_1m11vkb">
        <di:waypoint x="785" y="380" />
        <di:waypoint x="830" y="380" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1eahtb9_di" bpmnElement="Flow_1eahtb9">
        <di:waypoint x="760" y="405" />
        <di:waypoint x="760" y="470" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_03hvnx5_di" bpmnElement="Flow_03hvnx5">
        <di:waypoint x="1090" y="260" />
        <di:waypoint x="1120" y="260" />
        <di:waypoint x="1120" y="185" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_16hyvza_di" bpmnElement="Flow_16hyvza">
        <di:waypoint x="810" y="510" />
        <di:waypoint x="1020" y="510" />
        <di:waypoint x="1020" y="420" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_08g2s7f_di" bpmnElement="Flow_08g2s7f">
        <di:waypoint x="930" y="380" />
        <di:waypoint x="990" y="380" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_19945vu_di" bpmnElement="Flow_19945vu">
        <di:waypoint x="1040" y="340" />
        <di:waypoint x="1040" y="300" />
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
