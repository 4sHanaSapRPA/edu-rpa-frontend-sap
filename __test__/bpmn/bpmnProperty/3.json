{
  "processID": "Process_1jqmW5U",
  "processName": "David",
  "activities": [
    {
      "activityID": "StartEvent_0vr9as6",
      "activityType": "bpmn:StartEvent",
      "properties": {}
    },
    {
      "activityID": "Activity_110mni4",
      "activityType": "bpmn:Task",
      "properties": {}
    },
    {
      "activityID": "Event_14mwyv7",
      "activityType": "bpmn:EndEvent",
      "properties": {}
    },
    {
      "activityID": "Activity_0rcsfaj",
      "activityType": "bpmn:Task",
      "properties": {}
    },
    {
      "activityID": "Event_1mens5i",
      "activityType": "bpmn:StartEvent",
      "properties": {}
    },
    {
      "activityID": "Event_0wqbmaw",
      "activityType": "bpmn:EndEvent",
      "properties": {}
    },
    {
      "activityID": "Activity_1wojxqu",
      "activityType": "bpmn:Task",
      "properties": {
        "activityPackage": "Google Workspace",
        "serviceName": "Google Drive",
        "activityName": "Create folder",
        "arguments": {
          "Connection": "a",
          "Folder name": "a",
          "Parent Folder Path": "a",
          "Overwrite if folder exists": "a"
        }
      }
    },
    {
      "activityID": "Activity_04ooeee",
      "activityType": "bpmn:Task",
      "properties": {
        "activityPackage": "Google Workspace",
        "serviceName": "Google Drive",
        "activityName": "Create folder",
        "arguments": {
          "Folder name": "a",
          "Parent Folder Path": "a",
          "Overwrite if folder exists": "a",
          "Connection": "a"
        }
      }
    },
    {
      "activityID": "Flow_1fsrpl3",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Flow_16ge7ke",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Flow_0wg0km0",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Flow_1n04kn4",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Flow_0j2or04",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Activity_0q44pxx",
      "activityType": "bpmn:SubProcess",
      "properties": {
        "activityPackage": "Control",
        "serviceName": "For Each",
        "activityName": "For each ... in ...",
        "arguments": { "List": "${emails}", "Item": "${email}" }
      }
    },
    {
      "activityID": "Event_1slu96m",
      "activityType": "bpmn:StartEvent",
      "properties": {}
    },
    {
      "activityID": "Flow_04n9qzv",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    },
    {
      "activityID": "Flow_0qxc73l",
      "activityType": "bpmn:SequenceFlow",
      "properties": {}
    }
  ],
  "variables": []
}
