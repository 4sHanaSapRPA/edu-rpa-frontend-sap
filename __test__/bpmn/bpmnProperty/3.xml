<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1htzell" targetNamespace="http://bpmn.io/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="14.0.0">
  <bpmn:process id="Process_1jqmW5U" isExecutable="false">
    <bpmn:startEvent id="Event_1mens5i">
      <bpmn:outgoing>Flow_1n04kn4</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:endEvent id="Event_0wqbmaw">
      <bpmn:incoming>Flow_0j2or04</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:task id="Activity_1wojxqu" name="Create Folder">
      <bpmn:incoming>Flow_1n04kn4</bpmn:incoming>
      <bpmn:outgoing>Flow_04n9qzv</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_1n04kn4" sourceRef="Event_1mens5i" targetRef="Activity_1wojxqu" />
    <bpmn:task id="Activity_04ooeee">
      <bpmn:incoming>Flow_0qxc73l</bpmn:incoming>
      <bpmn:outgoing>Flow_0j2or04</bpmn:outgoing>
    </bpmn:task>
    <bpmn:sequenceFlow id="Flow_0j2or04" sourceRef="Activity_04ooeee" targetRef="Event_0wqbmaw" />
    <bpmn:subProcess id="Activity_0q44pxx">
      <bpmn:incoming>Flow_04n9qzv</bpmn:incoming>
      <bpmn:outgoing>Flow_0qxc73l</bpmn:outgoing>
      <bpmn:startEvent id="Event_1slu96m">
        <bpmn:outgoing>Flow_1u6qs6d</bpmn:outgoing>
      </bpmn:startEvent>
      <bpmn:endEvent id="Event_0mb64qa">
        <bpmn:incoming>Flow_1u6qs6d</bpmn:incoming>
      </bpmn:endEvent>
      <bpmn:sequenceFlow id="Flow_1u6qs6d" sourceRef="Event_1slu96m" targetRef="Event_0mb64qa" />
    </bpmn:subProcess>
    <bpmn:sequenceFlow id="Flow_04n9qzv" sourceRef="Activity_1wojxqu" targetRef="Activity_0q44pxx" />
    <bpmn:sequenceFlow id="Flow_0qxc73l" sourceRef="Activity_0q44pxx" targetRef="Activity_04ooeee" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1jqmW5U">
      <bpmndi:BPMNShape id="Event_1mens5i_di" bpmnElement="Event_1mens5i">
        <dc:Bounds x="72" y="-648" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_1wojxqu_di" bpmnElement="Activity_1wojxqu">
        <dc:Bounds x="280" y="-670" width="100" height="80" />
        <bpmndi:BPMNLabel />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_04ooeee_di" bpmnElement="Activity_04ooeee">
        <dc:Bounds x="1060" y="-940" width="100" height="80" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_0wqbmaw_di" bpmnElement="Event_0wqbmaw">
        <dc:Bounds x="1252" y="-888" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Activity_0q44pxx_di" bpmnElement="Activity_0q44pxx" isExpanded="true">
        <dc:Bounds x="580" y="-750" width="440" height="200" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_1slu96m_di" bpmnElement="Event_1slu96m">
        <dc:Bounds x="620" y="-668" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Event_0mb64qa_di" bpmnElement="Event_0mb64qa">
        <dc:Bounds x="932" y="-668" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="Flow_1u6qs6d_di" bpmnElement="Flow_1u6qs6d">
        <di:waypoint x="656" y="-650" />
        <di:waypoint x="932" y="-650" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_1n04kn4_di" bpmnElement="Flow_1n04kn4">
        <di:waypoint x="108" y="-630" />
        <di:waypoint x="280" y="-630" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0j2or04_di" bpmnElement="Flow_0j2or04">
        <di:waypoint x="1160" y="-900" />
        <di:waypoint x="1206" y="-900" />
        <di:waypoint x="1206" y="-870" />
        <di:waypoint x="1252" y="-870" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_04n9qzv_di" bpmnElement="Flow_04n9qzv">
        <di:waypoint x="380" y="-630" />
        <di:waypoint x="580" y="-630" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNEdge id="Flow_0qxc73l_di" bpmnElement="Flow_0qxc73l">
        <di:waypoint x="955" y="-750" />
        <di:waypoint x="955" y="-900" />
        <di:waypoint x="1060" y="-900" />
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
