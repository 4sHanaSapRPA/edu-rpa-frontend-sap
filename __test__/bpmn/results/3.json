{
  "block": [
    {
      "id": "Activity_0pkjxtm",
      "name": "Open Browser",
      "incomming": [
        "Flow_12ctyks"
      ],
      "outgoing": [
        "Flow_1ufgvsl"
      ]
    },
    {
      "id": "Activity_1536xo5",
      "name": "Get Title Assign to ${title}",
      "incomming": [
        "Flow_1ufgvsl"
      ],
      "outgoing": [
        "Flow_0c0pvmq"
      ]
    },
    {
      "start": "Gateway_0sc92or",
      "braches": [
        {
          "block": [
            {
              "id": "Activity_1gusysf",
              "name": "Log Screen",
              "incomming": [
                "Flow_1u5v65y"
              ],
              "outgoing": [
                "Flow_0acg68n"
              ]
            },
            {
              "id": "Activity_173a9c7",
              "name": "Click \"Dịch vụ Internet\"",
              "incomming": [
                "Flow_0acg68n"
              ],
              "outgoing": [
                "Flow_1r4gg47"
              ]
            }
          ]
        },
        {
          "block": [
            {
              "id": "Activity_0m9hqf4",
              "name": "Log Screen",
              "incomming": [
                "Flow_0q637w1"
              ],
              "outgoing": [
                "Flow_0xd1c08"
              ]
            },
            {
              "start": "Gateway_12fobzq",
              "braches": [
                {
                  "block": [
                    {
                      "id": "Activity_0p2itoz",
                      "name": "Click \"Thảo luận\"",
                      "incomming": [
                        "Flow_1m11vkb"
                      ],
                      "outgoing": [
                        "Flow_08g2s7f"
                      ]
                    }
                  ]
                },
                {
                  "block": [
                    {
                      "id": "Activity_072vuzz",
                      "name": "Break",
                      "incomming": [
                        "Flow_1eahtb9"
                      ],
                      "outgoing": [
                        "Flow_16hyvza"
                      ]
                    }
                  ]
                }
              ]
            },
            {
              "id": "Activity_1az9oha",
              "name": "Click \"Thảo luận\"",
              "incomming": [
                "Flow_08g2s7f",
                "Flow_16hyvza"
              ],
              "outgoing": [
                "Flow_19945vu"
              ]
            },
            {
              "id": "Activity_1juwgjw",
              "name": "Break",
              "incomming": [
                "Flow_19945vu"
              ],
              "outgoing": [
                "Flow_03hvnx5"
              ]
            }
          ]
        }
      ]
    },
    {
      "id": "Activity_1juwgjw",
      "name": "Break",
      "incomming": [
        "Flow_19945vu"
      ],
      "outgoing": [
        "Flow_03hvnx5"
      ]
    }
  ]
}