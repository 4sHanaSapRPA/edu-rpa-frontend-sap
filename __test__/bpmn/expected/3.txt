Activity_0pkjxtm - Open Browser
Activity_1536xo5 - Get Title Assign to ${title}
IF: Gateway_0sc92or
	Activity_1gusysf - Log Screen
	Activity_173a9c7 - Click "Dịch vụ Internet"
ELSE:
	Activity_0m9hqf4 - Log Screen - 2
	IF: Gateway_12fobzq
		Activity_0p2itoz - Click "Thảo luận"
	ELSE:
		Activity_072vuzz - Break

	Activity_1az9oha - Join Break
	Activity_1juwgjw - BreakOut

Activity_012ckfs - Close Browser